extensions/blocks/rating-star/index.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * External dependencies
+ */
+import { __, _x } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import edit from './edit';
+import save from './save';
+import { StarIcon, StarBlockIcon } from './icon';
+import './editor.scss';
+import './style.scss';
+
+export const name = 'rating-star';
+
+export const settings = {
+	title: 'Star Rating',
```

Comment:
The title of the block cannot be translated. Is this on purpose? Since it has already been referenced in support documents it is used untranslated everywhere.

Reply:
This was an oversight, nice catch. 

https://github.com/Automattic/jetpack/pull/14190